include/odp/api/spec/pool.h

```diff
@@ -214,6 +214,13 @@ typedef struct odp_pool_param_t {
 			    defined by pool capability pkt.max_uarea_size.
 			    Specify as 0 if no user area is needed. */
 			uint32_t uarea_size;
+
+			/** Headroom size in bytes. Number of bytes of
+			    space preceding packet data to reserve for
+			    use as headroom.
+			    Must not exceed the implementation defined
+			    ODP_PACKET_MAX_HEADROOM. */
```

Comment:
Will include these changes in the next version. Maybe we should rename it to default_headroom instead of min_headroom since technically this is value of headroom which the implementation will have by default and min_headroom can be zero.

Reply:
That sounds like a reasonable change.